samples/extensions/tensor_and_data_graph/simple_tensor_and_data_graph/README.adoc

```diff
@@ -0,0 +1,589 @@
+:source-highlighter: coderay
```

Comment:
I don't think this will work with the Antora site build and needs to be removed.